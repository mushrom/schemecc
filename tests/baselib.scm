(import (scheme base))

(print 1234)

(for-each write
    (cons #\a
      (cons #\b
        (cons #\c
              ()))))

(newline)
