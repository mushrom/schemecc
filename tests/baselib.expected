1234
#\a#\b#\c
1
